concrete/src/Cache/Adapter/ZendCacheDriver.php

```diff
@@ -87,12 +138,14 @@ protected function internalGetItem(&$normalizedKey, &$success = null, &$casToken
      */
     protected function internalSetItem(&$normalizedKey, &$value)
     {
-        /** @var Cache $cache  */
-        $cache = Core::make($this->cacheName);
-        $item = $cache->getItem('zend/'.$normalizedKey);
-
-        if ($result = $item->set($value, $this->cacheLifetime)) {
-            $item->save();
+        $cache = $this->getCache();
+        if ($cache === null) {
```

Comment:
For a cleaner code, I prefer to avoid assignments in `if` statements.
Indeed, if you read `if ($a = 2) {` you may think that's wrong, you have to go back to the code and check what the coder wanted.

About `getCacheKey`: fixed in upcoming commit that I'll add to this PR (#4444)